package.json

```diff
@@ -21,6 +21,15 @@
   "bugs": {
     "url": "https://github.com/cerner/terra-framework/issues"
   },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
```

Comment:
Should this be
```
"eslintConfig": {
  "extends": "eslint-config-terra"
},
```

Reply:
Good question! Either definition is valid, but eslint allows extends to obmit the `estlint-config` prefix. Here is the link to the docs: https://eslint.org/docs/user-guide/configuring#using-a-shareable-configuration-package.

Reply:
I feel like it should, I'm surprised using just `terra` works, though thats also what we use in terra-core. @emilyrohrbough what your thoughts on this?

Reply:
Eslint doesn't require the prefix to be included to successfully load the configuration. Eslint searched for the module `eslint-config-terra` when given `terra`. I am unopinionated to how we define it in the configuration.

Reply:
Let's keep it as is, TIL something new about ESLint configs 👍